detectors/aws/eks/detector.go

```diff
@@ -139,50 +139,34 @@ func (eksUtils eksDetectorUtils) fileExists(filename string) bool {
 	return err == nil && !info.IsDir()
 }
 
-// fetchString executes an HTTP request with a given HTTP Method and URL string.
-func (eksUtils eksDetectorUtils) fetchString(httpMethod string, URL string) (string, error) {
-	request, err := http.NewRequest(httpMethod, URL, nil)
-	if err != nil {
-		return "", fmt.Errorf("failed to create new HTTP request with method=%s, URL=%s: %w", httpMethod, URL, err)
-	}
-
-	// Set HTTP request header with authentication credentials
-	authHeader, err := getK8sCredHeader()
+// fetchString executes an HTTP Kubernetes client request with a given HTTP Method and URL string.
```

Comment:
Do you need to mention the API string now in this doc?

Reply:
Yes, I need to add that as well since the function signature has changed. Thanks!